_dashboards/dashboard/index.md

```diff
@@ -0,0 +1,163 @@
+---
+layout: default
+title: Creating dashboards
+nav_order: 30
+has_children: true
+---
+
+# Creating dashboards
+
+A dashboard combines multiple data visualizations into a single view. With the **Dashboard** application, you can:
+
+- Get started with pre-built panels and dashboards and customize display properties to create the dashboard you need.
+- Analyze, monitor, and alarm on metrics, logs, and traces across multiple data sources.
+- Create interactive dashboards and share them with anyone in your organization.
+
+<img src="{{site.url}}{{site.baseurl}}/images/dashboards/dashboard-index.png" alt="Single-view dashboard in OpenSearch Dashboards" width="600" height="300">
+
+Before you begin this tutorial, make sure you've installed OpenSearch and OpenSearch Dashboards and that you've connected to Dashboards at [http://localhost:5601](http://localhost:5601). The username and password are `admin`.
```

Comment:
A bit out of scope, but for the target user, I think it's a better assumption to expect that they'll either be doing it on their existing installation of OpenSearch Dashboards, or else on the playground: https://playground.opensearch.org/app/home.  Although neither is perfect, since sample data may not be available on the former, and the latter doesn't support saving.

Reply:
@joshuarrrr Please confirm the revised text addresses your feedback.